### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [testing-cabal/unittest-ext#48]
```
Ezio Melotti has added deprecation warnings to more methods in 
unittest.TestCase. (These were deprecated in the documentation but didn't have 
deprecation warnings.)

For unittest2 these probably need to be pending deprecation warning as 
deprecation warning is only silent by default in Python 2.7 and 3.2.

Ezio has said he will port this change to unittest2, this issue is just to 
track it. 

The corresponding Python issue is: http://bugs.python.org/issue9424
```

Original issue reported on code.google.com by `fuzzyman` on 25 Nov 2010 at 10:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `ezio.melotti` on 22 Feb 2011 at 1:25

**Comment by GoogleCodeExporter** [comment]
```
Done in http://hg.python.org/unittest2/rev/2ce0bdfa3bff
I left PendingDeprecationWarnings for all the functions for now, but I plan to 
turn them in DeprecationWarnings if/when I manage to fix issue #54.
```

Original comment by `ezio.melotti` on 16 Mar 2011 at 4:01
- Changed state: **Fixed**
